src/core/qgsconnectionpool.h

```diff
@@ -212,43 +223,46 @@ class QgsConnectionPoolGroup
 
     void onConnectionExpired()
     {
-      connMutex.lock();
+      connectionMutex.lock();
 
       QTime now = QTime::currentTime();
 
       // what connections have expired?
       QList<int> toDelete;
-      for ( int i = 0; i < conns.count(); ++i )
+      for ( int i = 0; i < connections.count(); ++i )
       {
-        if ( conns.at( i ).lastUsedTime.secsTo( now ) >= CONN_POOL_EXPIRATION_TIME )
+        if ( connections.at( i ).lastUsedTime.secsTo( now ) >= CONN_POOL_EXPIRATION_TIME )
           toDelete.append( i );
       }
 
       // delete expired connections
       for ( int j = toDelete.count() - 1; j >= 0; --j )
       {
         int index = toDelete[j];
-        qgsConnectionPool_ConnectionDestroy( conns[index].c );
-        conns.remove( index );
+        connectionDestroy( connections[index].connection );
+        connections.remove( index );
       }
 
-      if ( conns.isEmpty() )
+      if ( connections.isEmpty() )
         expirationTimer->stop();
 
-      connMutex.unlock();
+      connectionMutex.unlock();
     }
 
   protected:
 
-    QString connInfo;
-    QStack<Item> conns;
-    QList<T> acquiredConns;
-    QMutex connMutex;
-    QSemaphore sem;
+    QString connectionInfo;
```

Comment:
```suggestion
    QString mConnectionInfo;
```

all class members should be prefixed with `m`, this also applies to all members below.